components/parameter_items.py

```diff
@@ -141,6 +141,11 @@ def __init__(
     ):
         if param_key is None:
             param_key = name
+
+        # Convert 'options' to 'data' if present (for dmc.Select compatibility)
+        if "options" in kwargs:
```

Comment:
Is this needed due to incompatibility with `mlex_utils`? If so, could we update `mlex_utils`?